singer-io/tap-stripe
# TDL-14339 added configurable lookback window

# Description of change
- Added configurable lookback window and a default value in case not passed in the config.
- Added unittests for the same
- Increased the default lookback window size to 600(10 minutes) double than the current.

**NOTE**: The build is failing as we need to add a config parameter on the cci side.

# Manual QA steps
 - Check that the start_date starts from the configurable lookback window
 - Check that when lookback not configured, the tap takes a default value
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
